src/inspector/models/parts/partssettingsmodel.cpp

```diff
@@ -136,6 +138,11 @@ bool PartsSettingsModel::showTextLinkingOption() const
     return m_showTextLinkingOption;
 }
 
+bool PartsSettingsModel::isMasterScore() const
+{
+    return m_isMasterScore;
```

Comment:
There is AbstractInspectorModel::isMasterNotation()

Reply:
Ha! Thanks, didn't know